src/integration/java/io/pinecone/integration/dataplane/UpsertAndDescribeIndexStatsTest.java

```diff
@@ -0,0 +1,106 @@
+package io.pinecone.integration.dataplane;
+
+import io.pinecone.*;
+import io.pinecone.proto.*;
+import org.junit.jupiter.api.*;
+
+import static io.pinecone.helpers.BuildUpsertRequest.*;
+import static io.pinecone.helpers.GetOrCreateIndex.getOrCreateIndex;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class UpsertAndDescribeIndexStatsTest {
+    private static VectorServiceGrpc.VectorServiceBlockingStub blockingStub;
+    private static VectorServiceGrpc.VectorServiceFutureStub futureStub;
+    private static final int dimension = 3;
+
+    @BeforeAll
+    public static void setUp() throws IOException {
+        PineconeConnection connection = getOrCreateIndex(dimension);
+        blockingStub = connection.getBlockingStub();
+        futureStub = connection.getFutureStub();
+    }
```

Comment:
I think it may be good to set up an `@AfterAll` that uses `listIndexes` to handle deleting all indexes that may have been created during testing. We do that in integration tests for TS and I believe Python.

I think in this case given the usage of `getOrCreateIndex()` we expect to only be creating one index for use in this suite, but it may be good to go ahead and clean things up.

Reply:
The goal is to reuse the existing index for all data plane integration tests so I'm going to add a cron job to delete all indexes at every 24 hours